Composer/packages/client/src/components/AddRemoteSkillModal/SetAppId.tsx

```diff
@@ -27,20 +28,20 @@ type SetAppIdProps = {
   onNext: (targetName: string) => void;
   onGotoCreateProfile: () => void;
 };
+
 const getCreateProfileDescription = (botName, handleCreateProfile) => ({
   iconProps: {
     iconName: 'Error',
     color: SharedColors.orange20,
   },
   title: formatMessage(`Create a publishing profile for {botName}`, { botName }),
-  description: formatMessage(
-    'Your root bot must have an associated Microsoft App Id and Password to connect to a skill.'
-  ),
+  description: formatMessage('Your root bot must have an Azure publishing profile.'),
   link: {
     text: formatMessage('Create a publishing profile'),
     onClick: handleCreateProfile,
   },
 });
+
 const manifestUrl = {
```

Comment:
both objects containing formatMessage on lines 45 and 51 will not localize unless they are defined within the context of a component. To fix, convert both of them to functions:
```const getSomething = () => return the current object you have```